activities.el

```diff
@@ -332,6 +333,26 @@ Kills buffers that have only been shown in that activity's
 frame/tab."
   :type 'boolean)
 
+(defcustom activities-sort-function #'activities-sort-by-active-age
+  "A function to use to sort by when prompting for activities.
+If nil, no sorting will be applied.  The function should take two
+arguments, both activity names (strings).  It should return
+non-nil if the first activity should sort before the second.  By
+default, a function is used which sorts active activities first,
+and then by age since modification."
+  :type '(choice (const :tag "No sorting" nil)
+		 (function :tag "A specific function")))
```

Comment:
I think this could be a bit more idiomatic by describing the option in terms of the result rather than the predicate used to achieve it.  For example, naming the option `activities-sort-by`.  Then the type spec should include the default predicate, and the tag would be, e.g. "Age", so the user could choose to sort by "Age" or "custom predicate."

The docstring needn't describe the predicate beyond its arguments, I think, since that's just how a sort predicate works.